backend/Recodo.API/Controllers/UserController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Recodo.BLL.Services;
+using Recodo.Common.Dtos.User;
+using System.Threading.Tasks;
+
+namespace Recodo.API.Controllers
+{
+    [ApiController]
+    [Route("api/User")]
+    public class UserController : ControllerBase
+    {
+        private readonly AuthService _authService;
+        private readonly UserService _userService;
+
+        public UserController(AuthService authService, UserService userService)
+        {
+            _authService = authService;
+            _userService = userService;
+        }
+
+        [HttpPost("Update")]
+        public async Task<IActionResult> Update([FromForm] UpdateUserDTO userDTO, [FromForm] IFormFile avatar)
+        {
+            await _userService.UpdateUser(userDTO, avatar);
+            return NoContent();
+        }
+
+        [HttpPost("UpdatePassword")]
```

Comment:
```suggestion
        [HttpPost("Update-Password")]
```
The route will look like '.../api/user/updatepassword' with the current option. Even for the client, it isn't very friendly. Update in the same for other endpoints